src/Core/Scheduler/Providers/TileProvider.js

```diff
@@ -34,6 +26,7 @@ TileProvider.prototype.preprocessDataLayer = function preprocessLayer(layer, vie
 
     layer.level0Nodes = [];
     layer.onTileCreated = layer.onTileCreated || (() => {});
+    layer.object3d.invQuaternion = layer.object3d.quaternion.clone().inverse();
```

Comment:
`invQuaternion` is never updated. And IMHO it's a bad idea to introduce this concept: just call inverse() when you need it.
Or maybe rewrite the code doing this to avoid requiring a matrix inversion (in worldToLocal) + 2 quaternion.inverse() call.